services/authenticator/src/routes/landlord.js

```diff
@@ -120,6 +215,60 @@ landlordRouter.post('/signin', async (req, res) => {
     logger.error(exc);
     res.sendStatus(500);
   }
+};
+
+landlordRouter.post('/signin', async (req, res) => {
+  if (req.body.email) {
+    return await userSignIn(req, res);
+  } else if (req.body.clientId) {
+    return await applicationSignIn(req, res);
+  } else {
+    logger.info('login failed some fields are missing');
+    return res.status(422).json({ error: 'missing fields' });
+  }
+});
+
+landlordRouter.use('/appcredz', needAccessToken(config.ACCESS_TOKEN_SECRET));
+landlordRouter.use('/appcredz', checkOrganization());
+landlordRouter.post('/appcredz', async (req, res) => {
+  // ensure the user is administrator
+  if (req.user.role !== 'administrator') {
```

Comment:
I think it might be safer to not allow to create an appcredz from a call authenticated with an appcredz. Only an admin authenticated in the ui should be able to do it. WDYT?

Reply:
It's hard to anticipate all use-cases users may want. This would protect against creation of backdoors with a leaked application credentials but it will prevent automatic provisioning.
Maybe, we can:
- restrict appcredz creation as proposed for now,
- create a more fine-grained access control, with Create, Read, Edit, Delete permissions on each resources and profiles to have a RBAC

Reply:
Okay let's keep it like that and make something more fine-grain in future PRs